client/java/src/main/java/io/openlineage/client/utils/jdbc/OverridingJdbcExtractor.java

```diff
@@ -0,0 +1,56 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.client.utils.jdbc;
+
+import java.net.URISyntaxException;
+import java.util.Properties;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class OverridingJdbcExtractor extends GenericJdbcExtractor implements JdbcExtractor {
```

Comment:
I'm not sure if this logic should be in a a separated class, or in GenericJdbcExtractor

Reply:
Isn't it `JdbcWihPortExtractor` ?

Reply:
It both overrides schema and allows to set default port